.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npx lint-staged --relative 
+npx pretty-quick --staged
```

Comment:
Does this override the existing linted-staged? Ideally we would keep our existing hook and just add prettier

Reply:
Do you mean that by keeping both hooks in pre-commit file? 
so the content will be
`npx lint-staged --relative
npx pretty-quick --staged`